model/riscv_platform.sail

```diff
@@ -24,6 +24,12 @@ val elf_entry = pure {
   c: "elf_entry"
 } : unit -> int
 
+function PhysaddrStr (Physaddr(addr) : physaddr) -> string = {
+  if physaddrbits_len == 34
+  then BitStr(0b00 @ addr)
+  else BitStr(addr)
+}
```

Comment:
I think it would be nicer to use a generic function. Also `BitsStr()` is some old alias; the native function is `bits_str()` so I think we should use that style. How about this?

```
// Print a bit vector in hex. If it is not a multiple of 4
// bits in length zero extend it so it is (bits_str() prints
// bit vectors that aren't a multiple of 4 bits in binary).
function hex_bits_str forall 'n . (x : bits('n)) -> string =
  bits_str(zero_extend((3 - (('n + 3) % 4)) + 'n, x))
```

The name combines the existing `bits_str` and `hex_str`.

I have definitely wanted this function for other things while debugging!

Finally it should go in `prelude.sail` rather than here.